mibidata/tiff.py

```diff
@@ -46,7 +46,7 @@ def _cm_to_micron(arg):
 
 # pylint: disable=too-many-branches,too-many-statements
 def write(filename, image, sed=None, optical=None, ranges=None,
-          multichannel=True, write_float=False):
+          multichannel=True, write_dtype=None):
```

Comment:
Let's just call it `dtype`, not `write_dtype`? And maybe keep `write_float` and either give a deprecation warning or error in case a user tries it?

Reply:
Done. Raises an error since it would be hard to tell if `write_float` was actually specified or if it was set to the default `False` value.